Vagrantfile

```diff
@@ -117,11 +120,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.define "#{VM_NAME_PREFIX}dind-host" do |config|
       config.vm.box = kube_box[kube_os]["name"]
       config.vm.box_url = kube_box[kube_os]["box_url"]
-      config.vm.provision "shell", inline: "/vagrant/contrib/vagrant/provision-dind.sh"
-      config.vm.provision "shell", inline: "/vagrant/hack/dind-cluster.sh config-host"
-      config.vm.provision "shell", privileged: false, inline: "/vagrant/hack/dind-cluster.sh restart"
+      config.vm.provision "shell", inline: "#{sync_to}/contrib/vagrant/provision-dind.sh"
+      config.vm.provision "shell", inline: "#{sync_to}/hack/dind-cluster.sh config-host"
+      config.vm.provision "shell", privileged: false, inline: "#{sync_to}/hack/dind-cluster.sh restart"
       config.vm.hostname = "openshift-dind-host"
-      config.vm.synced_folder ".", "/vagrant", type: vagrant_openshift_config['sync_folders_type']
+      config.vm.synced_folder ".", "/vagrant", disabled: true
+      config.vm.synced_folder sync_from, sync_to, type: vagrant_openshift_config['sync_folders_type']
```

Comment:
The first line is disabling the default share to /vagrant as is already being done for the single vm case: 

https://github.com/openshift/origin/blob/master/Vagrantfile#L195